spec/integration/_dev/benchmark/deploy/tf/spec.yml

```diff
@@ -0,0 +1,26 @@
+spec:
+  additionalContents: false
+  contents:
+    - description: Terraform infrastructure definition
+      pattern: '^*.tf$'
+      type: file
+      required: true
+    - description: Terraform dependency lock file
+      pattern: ".terraform.lock.hcl"
+      type: file
+      required: true
+    - description: Terraform template files
+      pattern: '^*.tftpl$'
+      type: file
+      required: false
+    - description: Environment variables exposed for Terraform
+      type: file
+      name: "env.yml"
+      contentMediaType: "application/x-yaml"
+      required: false
+      $ref: "./env.spec.yml"
```

Comment:
Could it be used this specification that is already in place for `_dev/deploy/tf` ?